configs/DebugBoard/BoardConfig.h

```diff
@@ -41,6 +41,9 @@
 #define PIN_BUTTON_R3  8
 #define PIN_BUTTON_A1  26
 #define PIN_BUTTON_A2  9 // Not connected
+#define PIN_BUTTON_TURBO -1
```

Comment:
Can we add the Turbo LED pin in here as well with a default -1 and a note that it should be kept on GP25 for the time being.